torch/_torch_docs.py

```diff
@@ -7958,6 +7962,8 @@ def merge_dicts(*dicts):
 This is the second value returned by :meth:`torch.sort`.  See its documentation
 for the exact semantics of this method.
 
+See also :func:`torch.take_along_dim`.
```

Comment:
Let's remove these "See also"s for now. I know I asked for a reference, but I think we should elaborate more and I don't want to block this PR on those changes.